Makefile

```diff
@@ -34,4 +34,4 @@ test-cover: ## Launch test coverage and send it to coverall
 	$(ENV) ./scripts/test-coverage.sh
 
 release: ## Build release
-	docker run --rm -v $PWD:/go/src/github.com/Meetic/blackbeard -w /go/src/github.com/Meetic/blackbeard -e GITHUB_TOKEN -t goreleaser/goreleaser:latest release --rm-dist
+	docker run --rm -v PWD:/go/src/github.com/Meetic/blackbeard -w /go/src/github.com/Meetic/blackbeard -e GITHUB_TOKEN -t goreleaser/goreleaser:latest release --rm-dist
```

Comment:
are you sure of that ? is it working without $pwd ?

Reply:
It can be either $(PWD) or PWD, if you want to use a env base value (kinda like GITHUB_TOKEN on the same line), but $PWD does not work